lib/modules/platform/gitlab/index.ts

```diff
@@ -855,10 +855,39 @@ export async function updatePr({
     body.remove_labels = removeLabels;
   }
 
-  await gitlabApi.putJson(
-    `projects/${config.repository}/merge_requests/${iid}`,
-    { body },
-  );
+  const updatedPrInfo = (
+    await gitlabApi.putJson<{
+      iid: number;
+      source_branch: string;
+      title: string;
+      state: string;
+      created_at: string;
+    }>(`projects/${config.repository}/merge_requests/${iid}`, { body })
```

Comment:
Create a type for that generic. 

Can we use Zod here?

Reply:
Yes, I think this is necessary can we can start with Gitlab as the types are messy here. A task for another PR though as we need to also handle the other functions like createPr, getPrList, getPr, getBranchPr, and findPr.